integration_tests/steps/jira_issues_tests.py

```diff
@@ -113,6 +113,8 @@ def step_impl(context):
 
 @then('on every market slot there should be matching trades on grid and house markets')
 def check_matching_trades(context):
+    from integration_tests.steps.integration_tests import get_simulation_raw_results
+    get_simulation_raw_results(context)
     for time_slot, core_stats in context.raw_sim_data.items():
```

Comment:
Didn't we discuss to change this function to have a return value?

Reply:
We agreed to adopt this style from now on, there are lots of examples that I think we can keep them atm.